migrations/20170419122022_make-is_primary-not-nullable.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+exports.up = (knex) => (
+  knex.schema
+    .raw('UPDATE records SET is_primary = false where is_primary IS NULL')
+    .raw('ALTER TABLE records ALTER COLUMN is_primary SET NOT NULL')
+);
+
+exports.down = (knex) => (
+  knex.schema
+    .raw('ALTER TABLE records ALTER COLUMN is_primary DROP NOT NULL')
```

Comment:
Didn't throw an error due to the destructive change in `up`, because the `ALTER TABLE` is reversible. Didn't look like a good candidate for splitting the migration in two different files though.